src/cfnlint/maintenance.py

```diff
@@ -18,6 +27,8 @@
 
 LOGGER = logging.getLogger(__name__)
 
+REGISTRY_SCHEMA_ZIP = 'https://schema.cloudformation.us-east-1.amazonaws.com/CloudformationSchema.zip'
```

Comment:
- Is this documented somewhere (this is much more user-friendly than how I was querying the registry in my account)?
- Do we need to download this for multiple regions (the eu-west-1  version of this URL works)?

Reply:
* [documented here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-type-schemas.html)
* [`us-east-1` is almost always roughly a superset of all regions](https://github.com/aws-cloudformation/cfn-python-lint/issues/51#issuecomment-515735722), which is why `us-east-1` is used as the default for `cfn-lint` itself. I'm okay with just using `us-east-1` for additional constraints for now. Looks like [mostly just `AWS::GroundStation` constraints might be missed at the moment](https://github.com/ScriptAutomate/aws-cfn-resource-specs/blob/master/CHANGELOG.md#existing-resourcetypes-and-propertytypes-not-in-us-east-1)